pkg/apis/psmdb/v1/perconaservermongodbbackup_types.go

```diff
@@ -7,12 +7,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-
 // PerconaServerMongoDBBackupSpec defines the desired state of PerconaServerMongoDBBackup
 type PerconaServerMongoDBBackupSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
```

Comment:
Which command we need to run instead of `operator-sdk generate k8s`?

Reply:
e2e-tests/manifest will update all crds for latest version. to generate next version provide next version argument like
e2e-tests/manifest v1.13.0

Reply:
Let's replace the comment with this new command then. It'd be helpful for our contributors.